src/content/blog/ja/ai-driven-development-workflow.md

```
@@ -0,0 +1,68 @@
+pubDate: 2024-01-15
+author: "Terisuke"
+category: "ai"
+tags: ["AI", "開発効率", "Cursor", "CodeRabbit", "GitHub Copilot"]
+# image:
+#   url: "/images/blog/ai-driven-development.avif"
+#   alt: "AI駆動開発のイメージ"
+lang: "ja"
+featured: true
+---
+
+# AI駆動開発で生産性が7倍に：私の実体験
```

markdownlint-cli2 (0.17.2)

Multiple top-level headings in the same document
null

(MD025, single-title, single-h1)